packages/ui/src/components/editor/panels/Assets/container/index.tsx

```diff
@@ -70,7 +79,13 @@ type Category = {
   depth: number
 }
 
-const AssetsPreviewContext = createContext({ onAssetSelectionChanged: (props: AssetSelectionChangePropsType) => {} })
+const AssetsPreviewContext = hookstate({
```

Comment:
any reason we arent using hyperflux here? or making it a scoped useHookstate?

Reply:
Nope, switched it to a hookstate